src/main/java/com/checkmarx/flow/service/GitLabService.java

```diff
@@ -12,8 +12,8 @@
 import com.checkmarx.flow.exception.GitLabClientException;
 import com.checkmarx.flow.utils.HTMLHelper;
 import com.checkmarx.flow.utils.ScanUtils;
-import com.checkmarx.sdk.dto.sast.CxConfig;
 import com.checkmarx.sdk.dto.ScanResults;
+import com.checkmarx.sdk.dto.sast.CxConfig;
 import org.apache.commons.codec.binary.Base64;
 import org.json.JSONArray;
 import org.json.JSONException;
```

Comment:
What are the changes in this file? not clear

Reply:
optimization from IDEA